panel/src/components/Layout/Overlay.vue

```diff
@@ -12,7 +12,7 @@
 			@mousedown="close"
 		>
 			<k-loader v-if="loading" class="k-overlay-loader" />
-			<slot v-else :close="close" :isOpen="isOpen" />
+			<slot v-else :close="close" :is-open="isOpen" />
```

Comment:
Not sure what this change does TBH

Reply:
[vue/attribute-hyphenation](https://eslint.vuejs.org/rules/attribute-hyphenation.html)

Reply:
Yes, I understand that part. But I was wondering if the different syntax could break behavior.

Reply:
It shouldn't. As I understand it, the Vue compiler will handle pascal case from props and attribute case in templates. The compiler is case-tolerant, that's why it's working currently as well.